.env.example

```
@@ -14,7 +14,9 @@ CONCURRENT_REQUEST_LIMIT=2
 MAX_PARALLEL_PROCESSES=2
 
 # --- Database settings ---
+POSTGRES_VERSION=16-alpine
 POSTGRES_USER=user
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POSTGRES_USER key should go before the POSTGRES_VERSION key

(UnorderedKey)